cli/pipelines/main.ttl

```diff
@@ -77,3 +86,50 @@
                        code:link <node:barnard59-base#filter> ] ;
   code:arguments     ( [ code:link <file:../lib/output-filter#removeCsvwTriples> ;
                          a         code:EcmaScript ] ) .
+
+<#toDataset> a :Step;
+  code:implementedBy [ a code:EcmaScript;
+                       code:link <node:rdf-stream-to-dataset-stream/bySubject.js>
+                     ] .
+
+<#toObservation> a :Step;
+  code:implementedBy [ a code:EcmaScript;
+                       code:link <node:barnard59-rdf/cube.js#toObservation>
+                     ] ;
+  code:arguments [
+                   code:name "observer";
+                   code:value "https://ld.stadt-zuerich.ch/"
+                 ], [
+                      code:name "observations";
+                                code:value [
+                                             a code:EcmaScript ;
+                                             code:link <file:../lib/cube#getObservationSetId>
+                                         ]
+
+                    ], [
+                         code:name "observation";
+                         code:value "({ dataset, observations }) => ([...dataset][0].subject)"^^code:EcmaScript
+                       ] .
```

Comment:
This is the key part of the new pipeline steps.

The `observation` parameter simply uses the generated `aboutUrl` from csvw as the observation URI

The `observations` parameter is code to generate the `cube:observationSet` URI. I propose that the CSVW would implicitly add a mapping to output a `cc:cube` property. The `cube:observationSet` would simply be `${cubeId}/observation/`